ansible/edpm_provisioning.yml

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
could we move this file to under roles/foreman/tasks/main.yml and have this role part of RHOSO deployment flow?
or this is intended to be excluded from the deployment flow?